src/platform/sdl/sdl_window.cpp

```
@@ -99,9 +99,8 @@ namespace {
 }
 
 namespace moth_graphics::platform::sdl {
-    Window::Window(graphics::sdl::Context& context, std::string_view title, int width, int height)
-        : platform::Window(title, width, height)
-        , m_context(context) {
+    Window::Window(std::string_view title, int width, int height)
```

Clang (14.0.6)

[error] constructor does not initialize these fields: m_surfaceContext

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)